swagger/swagger-generator/generator-core/src/main/java/org/apache/servicecomb/swagger/extend/introspector/JsonPropertyIntrospector.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.swagger.extend.introspector;
+
+import org.springframework.util.StringUtils;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import io.swagger.jackson.SwaggerAnnotationIntrospector;
+
+public class JsonPropertyIntrospector extends SwaggerAnnotationIntrospector {
+
+  private static final long serialVersionUID = 4157263023893695762L;
+
+  @SuppressWarnings("deprecation")
```

Comment:
Is there another way to replace deprecated API?

Reply:
swagger internal logic invoke this API directly.

Reply:
There seems no other way to achieve our purpose, since swagger invoke `AbstractModelConverter#findEnumValue` explicitly in `io.swagger.jackson.ModelResolver#_addEnumProps`.

`Jackson` suggests that `findEnumValue` should be replaces by `findEnumValues`. Maybe after the swagger is upgraded, we can also achieve it in `findEnumValues`.

Reply:
ok